modules/test/playwright/tests/journal-web/pages/JournalPage.ts

```diff
@@ -54,7 +54,8 @@ export class JournalPage {
 	}
 
 	async fillArticleData(title: string, content: string) {
-		await this.articleTitleInput.fill(title);
+		await this.articleTitleInput.click();
```

Comment:
As ridiculous as this sounds, the problem with title only occurs when creating the web content on a site template. Having that in mind we may need to extract the click outside the fillArticleData() method and use this new clickArticleTitle() together with fillArticleData() only in our methods. This will secure that no other team removes our necessary click :D

Reply:
at the same time, awesome discovery, @jaimelr10 ! :D Your solution is way cleaner and more efficient than what I have initially proposed, well done! 👏🏻